iset.mm

```diff
@@ -145625,6 +145648,46 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
       XMWEWHWIWSMXOWRWTXAAXDNLXENLXFXHTACYBXBABYHXBXDXEXCUOVO $.
   $}
 
+  $( Cosine is less than one in the third and fourth quadrants.  (Contributed
+     by Jim Kingdon, 19-Mar-2024.) $)
+  cosq34lt1 $p |- ( A e. ( _pi [,) ( 2 x. _pi ) ) -> ( cos ` A ) < 1 ) $=
+    ( cpi c2 cmul co wcel cmin cneg ccos cfv c1 clt caddc cc cr cle wbr a1i cc0
+    pire cico wceq cxr w3a wb 2re remulcli rexri elico2 mp2an simp1bi recnd 2cn
+    mulcli subcld cosneg syl mulm1d oveq2d negsubd eqtrd fveq2d cz neg1z cosper
+    picn sylancl 3eqtr2d 0xr 0re pipos ltleii lbicc2 mp3an12i resubcld renegcld
+    simp3bi posdifd mpbid negsubdi2d breqtrrd ltled ax-1cn subdiri 2m1e1 oveq1i
+    cicc mulid2i eqtri oveq2i 3eqtr3ri simp2bi subled eqbrtrd elicc2i syl3anbrc
+    eqbrtrid cosordlem cos0 breqtrdi eqbrtrrd ) ABCBDEZUAEFZAXBGEZHZIJZAIJZKLXC
+    XFXDIJZAKHZXBDEZMEZIJZXGXCXDNFXFXHUBXCAXBXCAXCAOFZBAPQZAXBLQZBOFZXBUCFXCXMX
+    NXOUDUETXBCBUFTUGZUHBXBAUIUJZUKZULZXBNFXCCBUMVFUNRZUOXDUPUQXCXKXDIXCXKAXBHZ
+    MEXDXCXJYBAMXCXBYAURUSXCAXBXTYAUTVAVBXCANFXIVCFXLXGUBXTVDAXIVEVGVHXCXFSIJKL
+    XCSXESUCFBUCFXCSBPQZSSBWGEZFVIBTUHYCXCSBVJTVKVLRSBVMVNXCXEOFSXEPQXEBPQXEYDF
+    XCXDXCAXBXSXBOFXCXQRZVOVPZXCSXESOFXCVJRYFXCSXBAGEZXELXCXOSYGLQXCXMXNXOXRVQX
+    CAXBXSYEVRVSXCAXBXTYAVTZWAZWBXCXEYGBPYHXCXBBAYEXPXCTRXSXCXBBGEZBAPCKGEZBDEZ
+    XBKBDEZGEBYJCKBUMWCVFWDYLYMBYKKBDWEWFBVFWHZWIYMBXBGYNWJWKXCXMXNXOXRWLWQWMWN
+    SBXEVJTWOWPYIWRWSWTXA $.
+
+  $( Cosine is less than one between zero and ` 2 x. _pi ` .  (Contributed by
+     Jim Kingdon, 19-Mar-2024.) $)
+  cos02pilt1 $p |- ( A e. ( 0 (,) ( 2 x. _pi ) ) -> ( cos ` A ) < 1 ) $=
+    ( cc0 c2 cpi co wcel clt wbr c1 wa c3 cr cle adantr pire a1i rexri halfpire
+    cxr wb cmul cioo cdiv ccos cfv cico elioore simpr ltled w3a remulcli elioo2
+    0xr 2re simp3bi ad2antrr elico2 syl3anbrc cosq34lt1 syl simplr 3re recoscld
+    mp2an 0red 1red cosq23lt0 0lt1 lttrd wo 2lt3 2pos pm3.2i pipos ltdiv2 mp3an
+    3pos mpbi ltdivmul wi axltwlin mp3an12i mpi mpjaodan cicc 0re ltleii lbicc2
+    simp2bi elicc2i cosordlem cos0 breqtrdi crp pirp rphalflt ax-mp ) ABCDUAEZU
+    BEFZDCUCEZAGHZAUDUEZIGHZADGHZWSXAJZDAGHZXCAKWTUAEZGHZXEXFJZADWRUFEFZXCXIALF
+    ZDAMHZAWRGHZXJXEXKXFWSXKXAABWRUGZNZNZXIDADLFZXIOPXPXEXFUHUIWSXMXAXFWSXKBAGH
+    ZXMBSFZWRSFZWSXKXRXMUJTUMWRCDUNOUKQZBWRAULVDZUOUPXQXTXJXKXLXMUJTOYADWRAUQVD
+    URAUSUTXEXHJZAWTXGUBEFZXCYCXKXAXHYDXEXKXHXONWSXAXHVAXEXHUHWTSFXGSFYDXKXAXHU
+    JTWTRQXGKWTVBRUKZQWTXGAULVDURYDXBBIYDAAWTXGUGVCYDVEYDVFAVGBIGHYDVHPVIUTXEDX
+    GGHZXFXHVJZDKUCEWTGHZYFCKGHZYHVKCLFZBCGHZJKLFZBKGHZJZXQBDGHZJYIYHTYJYKUNVLV
+    MYLYMVBVQVMZXQYOOVNVMCKDVOVPVRXQWTLFZYNYHYFTORYPDWTKVSVPVRXQXGLFXEXKYFYGVTO
+    YEXODXGAWAWBWCWDWSXDJZXBBUDUEIGYRBABBDWEEZFZYRXSDSFBDMHYTUMDOQBDWFOVNWGBDWH
+    VPPYRXKBAMHADMHAYSFWSXKXDXNNZYRBAYRVEUUAWSXRXDWSXKXRXMYBWINZUIYRADUUAXQYROP
+    WSXDUHUIBDAWFOWJURUUBWKWLWMWSWTDGHZXAXDVJZDWNFUUCWODWPWQYQXQWSXKUUCUUDVTROX
+    NWTDAWAWBWCWD $.
+
```

Comment:
I think these theorems should be added to set.mm, too.

Reply:
OK with me. I have revised this pull request to add both `cos02pilt1` and `cosq34lt1` .